packages/@aws-cdk/aws-codecommit/lib/repository.ts

```diff
@@ -1,10 +1,23 @@
+import * as notifications from '@aws-cdk/aws-codestarnotifications';
 import * as events from '@aws-cdk/aws-events';
 import * as iam from '@aws-cdk/aws-iam';
 import { IResource, Lazy, Resource, Stack } from '@aws-cdk/core';
 import { Construct } from 'constructs';
 import { CfnRepository } from './codecommit.generated';
 
-export interface IRepository extends IResource {
+/**
+ * Additional options to pass to the notification rule
```

Comment:
Docs should be complete sentences, with fullstops.

Same comment everywhere in the PR.

Reply:
Still not resolved I see 🙂.